bhop.s

```diff
@@ -2036,6 +2114,43 @@ check_for_inactive:
         rts
 .endproc
 
+.proc trigger_sample
+; from Channels2A03.cpp:
+; mEnabled = true;
+; mTriggerSample = true;
+
+; // If mRetriggerPeriod != 0, this initializes retriggering. Otherwise reset mRetriggerCtr.
+; queueSample();
+        lda channel_status + DPCM_INDEX
+        and #($FF - CHANNEL_SUPPRESSED)
```

Comment:
Even if the tracker does this, bhop should not. Nothing except a user-initiated call to `bhop_unmute` should clear the SUPPRESSED flag. The user has full control over this state, and the music driver must respect that and leave it alone.

Reviewing how the tracker uses mEnabled in code, I'm wondering if it more closely parallels bhop's MUTED state, rather than its SUPPRESSED state.

Reply:
what are the functions of `CHANNEL_MUTED` and `CHANNEL_SUPPRESSED` that make them distinct?